_posts/randy/2022-07-24-how-video-takes-all-day.md

```diff
@@ -0,0 +1,189 @@
+---
+layout: post
+title: "How Video Takes All Day"
```

Comment:
I would suggest we do "How Can a 10 Second Video Take All Day? Part 2" or something of the likes.  Makes it clearer that it is a direct followup.  Otherwise these titles sound too similar, but different.

Reply:
Along the same lines, I'd suggest add a link to this article from the first part so that once someone finishes the first part they are recommended to keep reading on to part 2

Reply:
agreed 2